akka-http/src/main/scala/akka/http/scaladsl/server/directives/FileUploadDirectives.scala

```diff
@@ -97,7 +100,8 @@ trait FileUploadDirectives {
             val fileInfo = FileInfo(part.name, part.filename.get, part.entity.contentType)
             val dest = destFn(fileInfo)
 
-            part.entity.dataBytes.runWith(FileIO.toPath(dest.toPath)).map { _ ⇒
+            part.entity.dataBytes.runWith(FileIO.toPath(dest.toPath)).map { ioResult ⇒
```

Comment:
`part.entity.dataBytes.runWith(FileIO.toPath(dest.toPath)).transform(_.status.map(_ => (fileInfo, dest)))`

Reply:
@viktorklang 
I tried it.
But that failed with ci.